P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1326 @@
+          WhatsApp or RocketChat.</item>
+      </list></p>
+    <p>Three of the four basic CMC units described above can be represented with models that are
+      described elsewhere in the TEI Guidelines:</p>
+    <table>
+      <row rend="underline">
+        <cell>CMC unit</cell>
+        <cell>Type of corpus data</cell>
+        <cell>TEI P5 element</cell>
+      </row>
+      <row>
+        <cell>utterance</cell>
+        <cell>transcription of speech</cell>
+        <cell>
+          <gi>u</gi>
+        </cell>
+      </row>
+      <row>
+        <cell>bodily activity</cell>
+        <cell>textual description</cell>
+        <cell>
+          <gi>kinesic</gi>
+        </cell>
+      </row>
+      <row>
+        <cell>onscreen activity</cell>
+        <cell>textual description</cell>
+        <cell>
+          <gi>incident</gi>
+        </cell>
+      </row>
+    </table>
+    <p>The <gi>u</gi>, <gi>kinesic</gi>, and <gi>incident</gi> elements are not limited to CMC, but
+      apply to encode textual transcriptions of spoken turns and CMC data about bodily activity and
+      onscreen activity. The CMC unit <term>post</term>, which is specific to
+      CMC, is introduced in <ptr target="#CMCcmcpost"/>.</p>
+  </div>
+  <div type="div2" xml:id="CMCcmc">
+    <head>Encoding Unique to CMC</head>
+    <p>This section describes elements, attributes, and models which are unique to CMC and the TEI
+      CMC module.</p>
+    <div xml:id="CMCcmcpost">
+      <head>CMC Posts</head>
+      <p>While the concept of a <term>post</term> is not unique to
+      computer-mediated communication (ask anyone who has posted a
+      <q>lost cat</q> sign in the local market), this chapter concerns
+      itself only with postings within a framework of a CMC system.
+      Thus the element <gi>post</gi> is unique to the encoding of
+      computer-mediated communication (CMC).
+      <specList>
+        <specDesc key="post"/>
+      </specList> Posts occur in a broad range of written CMC genres,
+        including (but not limited to) messages in chats and WhatsApp dialogues, tweets in X
+        (Twitter) timelines, comments on Facebook pages, posts in forum threads, and comments or
+        contributions to discussions on Wikipedia talk pages or in the comment sections of
+        weblogs.</p>
+      <p>Posts can be either written or spoken: <list>
+          <item><emph rend="bold">written</emph> or <emph rend="bold">multimodal posts</emph>: In
+            the majority of CMC technologies posts are composed as stretches of text using a
+            keyboard or speech-to-text conversion software in an entry form on the screen. In
+            many cases the technology allows authors to include or embed graphics (emojis or
+            images), video files, and hyperlinks into their posts.</item>
+          <item><emph rend="bold">audio posts</emph>: A growing number of CMC technologies, e.g.
```

Comment:
spoken posts instead of audio posts according to the list intro?

Reply:
Some audio posts might be nonverbal, as in musical or sound effects. So I think audio is more inclusive.

Reply:
True. But we should consider how we can remain consistent. The graphic says "spoken (audio post)", the `<post>` description "a written (or spoken) contribution", and the list intro is "Posts can be either written or spoken:"